feature/interface/aggregate/otg_tests/aggregate_forwarding_viable_test/aggregate_forwarding_viable_test.go

```diff
@@ -380,9 +380,9 @@ func (tc *testArgs) configureATE(t *testing.T) {
 		autoNegotiate.SetRsFec(false)
 	}
 
-	dstDev := tc.top.Devices().Add().SetName(agg.Name())
+	dstDev := tc.top.Devices().Add().SetName(agg.Name() + ".dev")
 	dstEth := dstDev.Ethernets().Add().SetName(ateDst.Name + ".Eth").SetMac(ateDst.MAC)
-	dstEth.Connection().SetLagName(agg.Name())
+	dstEth.Connection().SetChoice(gosnappi.EthernetConnectionChoice.LAG_NAME).SetLagName(agg.Name())
```

Comment:
Same comment as above, Gosnappi automatically sets the choice when calling `SetLagName`

Reply:
Removed.